portal/src/main/scala/cromwell/pipeline/datastorage/dao/entry/UserEntry.scala

```diff
@@ -33,8 +33,6 @@ trait UserEntry {
 
   def addUserAction(user: User) = (users.returning(users.map(_.userId))) += user
 
-  def deactivateUserByEmail(email: String) = users.filter(_.email === email).map(_.active).update(false)
```

Comment:
Why it's removed?

Reply:
@ilyaonishenko I thought it would be better to remove a deactivationByEmail feature completely but last time forgot about this method in UserEntry. I'm not sure if we'll need it in the future

Reply:
Ok, good.

For the future: best practice is to do all changes in its own branch or commit. Removing user methods should not be done in `gitClient` branch

Reply:
Ok, I've got it.
Looks logical